src/js/jsx/shared/Datalist.jsx

```diff
@@ -202,10 +207,12 @@ define(function (require, exports, module) {
             switch (event.key) {
             case "ArrowUp":
                 select.selectPrev();
+                event.preventDefault();
```

Comment:
Without it, the cursor in the text input jumps around while you are arrowing up/down.